### PR TITLE
Added release notes for k8s-nmstate

### PR DESCRIPTION
**Pull request by johnwilkins**
Fixes: [telcodocs-179](https://issues.redhat.com/browse/telcodocs-179)

@ahardin-rh 

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

**Comment by ahardin-rh** [comment]
This PR is dependent upon https://github.com/openshift/openshift-docs/pull/29078

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -166,6 +166,17 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
 
 For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode, the `app-netutil` library now provides the following functions: `GetCPUInfo()`, `GetHugepages()`, and `GetInterfaces()`. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-app-netutil_about-sriov[DPDK library for use with container applications].
 
+[id="ocp-4-7-networking-k8s-nmstate"]
+==== Kubernetes NMState (Technology Preview)
+
+{product-title} 4.7 provides post-installation state-driven network configuration on the secondary network interfaces of cluster nodes using `kubernetes-nmstate` as a technology preview. For more information, see
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:172]: as a Technology Preview feature

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -166,6 +166,17 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
 
 For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode, the `app-netutil` library now provides the following functions: `GetCPUInfo()`, `GetHugepages()`, and `GetInterfaces()`. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-app-netutil_about-sriov[DPDK library for use with container applications].
 
+[id="ocp-4-7-networking-k8s-nmstate"]
+==== Kubernetes NMState (Technology Preview)
```
Comment (jboxman) [inline release_notes/ocp-4-7-release-notes.adoc:170]: Isn't this an Operator? Shouldn't it be Kubernetes Nmstate Operator, with the proper name consistent with upstream usage?

Comment (jboxman) [reply release_notes/ocp-4-7-release-notes.adoc:170]: Looks like I'm incorrect, it is _Kubernetes NMState Operator_.

**Pull request closed without merging**